jupyterlab_caip_optimizer/cloudbuild-release.yaml

```diff
@@ -0,0 +1,33 @@
+substitutions:
```

Comment:
I think we're going to implement CI/CD with GitHub actions, so you can remove this if you like. Most users won't have permissions to use the `deeplearning-platform-ui-public` bucket anyways.